INSTALL.rst

```diff
@@ -89,13 +89,16 @@ zlib
 
 boost
 """""
-- 1.62.0-1.64.0 (``program_options``, ``regex`` , ``filesystem``, ``system``, ``math``, ``serialization`` and header-only libs, optional: ``fiber`` with ``context``, ``thread``, ``chrono``, ``atomic``, ``date_time``)
-- download from `http://www.boost.org <http://sourceforge.net/projects/boost/files/boost/1.62.0/boost_1_62_0.tar.gz/download>`_
+- 1.62.0 - 1.68.0 (``program_options``, ``regex`` , ``filesystem``, ``system``, ``math``, ``serialization`` and header-only libs, optional: ``fiber`` with ``context``, ``thread``, ``chrono``, ``atomic``, ``date_time``)
```

Comment:
Help wanted: we should verify that boost 1.67.0 and 1.68.0 work for us, especially with nvcc

Reply:
1.67.0 seems to be working, 1.68.0 is problematic (see alpaka link below)